code/modules/mob/living/silicon/ai/freelook/eye.dm

```diff
@@ -47,18 +100,18 @@
 		return ai.client
 	return null
 
-/mob/camera/aiEye/proc/RemoveImages()
-	var/client/C = GetViewerClient()
-	if(C && use_static)
-		C.images -= GLOB.cameranet.obscured
-
 /mob/camera/aiEye/Destroy()
 	if(ai)
 		ai.all_eyes -= src
 		ai = null
 	for(var/V in visibleCameraChunks)
 		var/datum/camerachunk/c = V
 		c.remove(src)
+	GLOB.aiEyes -= src
+	if(ai_detector_visible)
+		var/datum/atom_hud/ai_detector/hud = GLOB.huds[DATA_HUD_AI_DETECT]
+		hud.remove_from_hud(src)
+		QDEL_LIST(hud_list[AI_DETECT_HUD])

```

Comment:
This will resolve to `hud_list[AI_DETECT_HUD].Cut()` which is actually a : operator. No dice

Reply:
It's pretty silly to have to worry about implicit : operators in the expansion of defines